app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelMain.tsx

```diff
@@ -1,44 +1,73 @@
 import React, { ReactNode } from 'react';
 import { cx } from '@emotion/css';
 import Tabs, { Tab, TabList, TabPanel } from '@atlaskit/tabs';
-import { connectionPanelMainContainer, connectionPanelMainTabs, setUpGuideContainer } from './ConnectionPanel.styles';
+import {
+	connectionPanelMainContainer,
+	connectionPanelMainConnectedTabs,
+	connectionPanelMainNotConnectedTabs, setUpGuideContainer
+} from './ConnectionPanel.styles';
 import { ConnectedState } from '../StatusLabel/StatusLabel';
 import { NotConnectedState } from './NotConnectedState';
+import { JenkinsServer } from '../../../../src/common/types';
+import { ConnectedJenkinsServers } from './ConnectedJenkinsServers';
 import { SetUpGuide } from './SetUpGuide';
 
+// TODO - remove ? for children and connectedState once set up guide is merged
+type PanelProps = {
+	children?: ReactNode,
+	connectedState?: ConnectedState,
+	'data-testid'?: string
+};
+
 export const Panel = ({
 	children,
+	connectedState,
 	'data-testid': testid
-}: {
-	children: ReactNode;
-	'data-testid'?: string;
-}) => (
-	<div className={cx(testid === 'setUpGuidePanel'
-		? setUpGuideContainer : connectionPanelMainTabs)} data-testid={testid }>
-		{children}
-	</div>
-);
+}: PanelProps) => {
+	let className;
+
+	if (testid === 'setUpGuidePanel') {
```

Comment:
using the testid outside of tests feels strange, but if thats what needed 🤷

Reply:
lol you know this was in the PR you approved yesterday? 😆 It's just up as green coz I had to refactor to make sure this supported both the ConnectedState tab and the SetUpGuide one. I agree but think our only other alternative is to pass a isSetUpGuide prop from Panel component of SetUpGuide. Thoughts?